### PR TITLE
assign tas and instructors edge cases and corrections

### PR DESCRIPTION
**Pull request by rachaelcharles**
Reset allowances based on the original project description from Rock
Added missing tests and updated tests to include new functionality

### CODE REVIEW

File: TAScheduler/domain/CommandHandler.py [hunk 1]
```diff
@@ -209,28 +209,30 @@ def _CreateLabHandler(self, cmd: [str]):
         return 'Lab created'
 
     def _AssignCourseTAHandler(self, cmd: [str]):
-        if self.currentUser is None or not self.currentUser.RoleIn(Role.Administrator, Role.Supervisor):
+        if self.currentUser is None or not self.currentUser.RoleIn(Role.Instructor, Role.Supervisor):
             return 'Must be logged in as an Administrator or Supervisor'
```
Comment (dakotamaker) [inline TAScheduler/domain/CommandHandler.py]: This will need to be changed

File: TAScheduler/domain/CommandHandler.py [hunk 2]
```diff
@@ -209,28 +209,30 @@ def _CreateLabHandler(self, cmd: [str]):
...
-            return 'Must be logged in as an Administrator or Supervisor'
+        if self.currentUser is None or not self.currentUser.RoleIn(Role.Supervisor):
+            return 'Must be logged in as a Supervisor'
         if len(cmd) != 2:
             return ErrorMessages.INVALID_NUM_OF_ARGUMENTS
         try:
             c = Course.objects.get(course_name=cmd[0])
         except Exception:
-            return 'This course does not exist'
+            return 'A course with that name does not exist'
```
Comment (dakotamaker) [inline TAScheduler/domain/CommandHandler.py]: Make sure if you're changing error messages that you also change tests to reflect them. Not sure if you've done it yet, just a general reminder to everyone to run "python manage.py tests" in terminal and verify all tests are passing before committing any PRs

File: TAScheduler/domain/CommandHandler.py (hunk 1 above)
Comment (rachaelcharles) [reply TAScheduler/domain/CommandHandler.py]: Shoot -- yeah, my tests were passing the lab stuff (instead of course) to command handler, that's why they didn't catch this.  Got it.  The rest of the error messages should be okay, but double checking now.
